runtime/doc/options.txt

```diff
@@ -6519,10 +6518,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	|+path_extra| feature}
 	The |tagfiles()| function can be used to get a list of the file names
 	actually used.
-	If Vim was compiled with the |+emacs_tags| feature, Emacs-style tag
```

Comment:
Okay, sorry but my reading comprehension needs some work. Given that a lot of the Emacs stuff is in the immediate vicinity of of the `'tags'` stuff, it can probably be squashed together.

Besides that, LGTM (truthfully, I swear it :smirk:).